### DISCUSSION TITLE
Captions in timeline in FCP to Resolve and Back possible?

### DISCUSSION THREAD

**Discussion by tangierc** [CommandPost/FCPCafe#346] [Final Cut Pro Questions]
Hey FCP community. Has anyone been able to move a timeline with captions from Final Cut Pro to Resolve and back? I am working on a project in FCP that has captions, but someone who is going to edit them needs them in Resolve because they don't own Final Cut Pro. Then the intent is to get the Resolve timeline back. 

I am looking for any tools that may make this possible. I can't import the .scc file from FCP into Resolve. Importing an FCP timeline into Resolve doesn't bring over my SRT or CEA-608 versions of the captions on my FCP timeline. I need them to be in sequence on the timeline, not as individual captions clips, which do import into Resolve, but they don't actually show anything and aren't viewable from the Resolve media pool or timeline.

**Comment by TomThistlethwaite** [comment]
Hey Tangier,

Maybe I'm not quite understanding it, but could you export out your timeline as an XML and then export out a separate SRT with the captions from FCPX? Bring the XML timeline into Resolve and then import the SRT and lay it down. And once all done, do the same in reverse? As a slight aside, the guys at editingtools.io have this amazing converter:

https://en.editingtools.io/subtitles/

Cheers,

Tom
